service/controller/chart/v1/resource/status/create.go

```diff
@@ -83,6 +83,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		Reason:     reason,
 		Release: v1alpha1.ChartStatusRelease{
 			LastDeployed: v1alpha1.DeepCopyTime{releaseHistory.LastDeployed},
+			Revision:     releaseHistory.Revision,
```

Comment:
We now set the revision of the helm release. This is very useful for trouble shooting as you can compare against `helm history` and the release configmap.